src/markdown/lexicon/core-components/badges/index.mdx

```diff
@@ -3,7 +3,7 @@ title: 'Badges'
 titleLabel: 'Clay'
 titleLabelLink: 'https://clayui.com/docs/components/badge.html'
 figmaLink: 'https://www.figma.com/file/02DVhuHVTDbzaMjNM7IUKWp0/lexicon?node-id=6012%3A45959'
-docLink: 'https://drive.google.com/drive/folders/0B-KKkYVQA8saNzRRRDhUSlhRR0k?usp=sharing'
+docLink: 'https://docs.google.com/document/d/1PNudTqmS1JCSJwMt_h2CA9QobplxIDfwF7tZsjUpavw/edit?usp=sharing'
```

Comment:
This link points out to tables and not to badges. Wrong link, please check again